sdk/resourcemanager/Azure.ResourceManager/src/ManagementGroup/Generated/Models/ManagementGroupNameAvailabilityContent.cs

```diff
@@ -15,7 +15,6 @@ public partial class ManagementGroupNameAvailabilityContent
         /// <summary> Initializes a new instance of ManagementGroupNameAvailabilityContent. </summary>
         public ManagementGroupNameAvailabilityContent()
         {
-            ResourceType = "Microsoft.Management/managementGroups";
```

Comment:
Why is this removed?

Reply:
Because the "const" here is optional. We did not assign values for optional constants. They will have "null" as their default values.

Reply:
Yeah - I have the same concern and I have to update the related test cases to ensure they could pass.
We could make a change for "optional constants", we always make it default to its value instead of null